backend/src/Edna/Analysis/FourPL.hs

```diff
@@ -49,8 +63,49 @@ instance ToSchema Params4PL where
   declareNamedSchema Proxy =
     declareNamedSchema @Params4PLTuple Proxy
 
+type AnalysisResult = Either Text Params4PL
+
+instance Buildable (AnalysisResult) where
+  build = build . (coerce :: AnalysisResult -> Either Text Params4PL)
```

Comment:
Why do we need this `coerce`? Looks like `. id`.

Reply:
I don't understand why I need this instance \
Looks like it do nothing as its `type`, not `newtype` \
But without this instance it doesn't compile

Reply:
> I don't understand why I need this instance 

What do you mean? You need this instance because you have
```
instance Buildable (ForResponseLog AnalysisResult) where
  build = buildForResponse
```

and this `buildForResponse` requires `instance Buildable AnalysisResult` which you need to define.

Reply:
But `AnalysisResult` is just type synonym

Reply:
> But `AnalysisResult` is just type synonym

So what? There no `instance Buildable (Either a b)` if that's what you mean.

Reply:
I thought that it is and didn't understand what is doing on